deploy/crds/atlassian.com_cyclenoderequests_crd.yaml

```diff
@@ -44,6 +44,8 @@ spec:
               of an object. Servers should convert recognized schemas to the latest
               internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
+          clusterName:
```

Comment:
Where does this field come from? Why is it only in the `CycleNodeRequest` resource?

Reply:
It was part of the operator-sdk render, `make generate-crds`